### ISSUE TITLE
Backup tests are broken

### ISSUE THREAD

**Issue by AlCalzone** [ioBroker/ioBroker.js-controller#329]
The check for the exit code expects 26 to be falsy:
https://travis-ci.org/ioBroker/ioBroker.js-controller/jobs/489955280#L3150

**Comment by AlCalzone** [comment]
@foxriver76 Mind taking another look at this? It seems this is related to https://github.com/ioBroker/ioBroker.js-controller/pull/322, but I'm not sure what broke. Somehow the PR did not fail travis.

**Comment by foxriver76** [comment]
I will look into, it's failing since https://github.com/ioBroker/ioBroker.js-controller/commit/899caa2d399c0214bd4d92d78cff61477b4cba73

**Comment by AlCalzone** [comment]
Ok then its most likely this:
```diff
- const tmpDir = pathLib.normalize(__dirname + '/../../tmp');
- const bkpDir = pathLib.normalize(__dirname + '/../../backups');
+ const tmpDir = pathLib.normalize('../../tmp');
+ const bkpDir = pathLib.normalize('../../backups');
```

While relative paths are fine for `require` calls, which are always relative to `__dirname`, the paths here are used to read files, which are (most likely) relative to `process.cwd`.

**Comment by AlCalzone** [comment]
Looks good, the tests are going green again.

**Comment by AlCalzone** [comment]
Fixed by #330 
Why does the automatic closing of referenced issues not work here? :thinking:
